rpcs3/Emu/NP/np_handler.h

```diff
@@ -274,9 +276,23 @@ namespace np
 			SceNpMatching2ContextId ctx_id;
 			vm::ptr<SceNpMatching2RequestCallback> cb;
 			vm::ptr<void> cb_arg;
+			SceNpMatching2Event event_type;
+
+			void queue_callback(u32 req_id, u32 event_key, s32 error_code, u32 data_size) const
+			{
+				if (cb)
+				{
+					sysutil_register_cb([=, *this](ppu_thread& cb_ppu) -> s32
```

Comment:
isn't the copy of this redundant with [=] ?

Reply:
Copies of this need to be explicit to avoid warnings.